src/components/Tree/expandCollapseButton.js

```diff
@@ -14,22 +14,18 @@ function getIcon(isExpanded) {
 }
 
 export default function ExpandCollapseButton(props) {
-    const { hasChildren, isExpanded, isLoading, onClick } = props;
+    const { isExpanded, isLoading, onClick } = props;
     if (isLoading) {
         return (
             <SpinnerContainer>
                 <Spinner variant="brand" size="x-small" />
             </SpinnerContainer>
         );
     }
-    if (hasChildren) {
-        return <Button size="x-small" icon={getIcon(isExpanded)} onClick={onClick} tabIndex={-1} />;
-    }
-    return null;
+    return <Button size="x-small" icon={getIcon(isExpanded)} onClick={onClick} tabIndex={-1} />;
```

Comment:
This change implies that the expand collapse button will render always, we should avoid rendering it if not needed.